src/Validator.php

```diff
@@ -167,6 +167,15 @@ public static function validateDoctrineFieldName(string $name, ManagerRegistry $
         return $name;
     }
 
+    public static function validateEmailAddress(string $email): string
```

Comment:
method name could be as is or just `emailAddress`. As a whole, the class has functions like `validate..` and `notBlank`. Although `Validator::class` naming conventions are outside the scope of this PR, may be worth considering which name would be appropriate for this method.